pkg/trace/sampler/remote_rates.go

```diff
@@ -59,7 +59,15 @@ func newRemoteRates(client config.RemoteClient, maxTPS float64, agentVersion str
 }
 
 func (r *RemoteRates) onUpdate(update config.SamplingUpdate) {
-	log.Debugf("fetched config version %d from remote config management", update.Version)
+	// TODO: We don't have a version per product, yet. But, we will have it in the next version.
+	// In the meantime we will just use a version of one of the config files.
+	var version uint64
+	for _, v := range update.Configs {
```

Comment:
We can update `SamplingUpdate` as following
```
type SamplingUpdate struct {
	Version string
	Rates   []pb.APMSampling
}
```
then move this logic to `cmd/trace-agent/config/remote_client.go`.

At this stage, we don't really need to know if there were multiple files or not.
@gbbr any thoughts?

Reply:
No strong opinions. As you prefer.